lib/ae_mdw_web/websocket/socket_handler.ex

```diff
@@ -1,4 +1,6 @@
 defmodule AeWebsocket.Websocket.SocketHandler do
+  @moduledoc false
+  # credo:disable-for-this-file
```

Comment:
If this branch is rebased, this isn't really needed anymore

Reply:
the base branch wasn't updated, done